app/(app)/jobs/create/_client.tsx

```diff
@@ -0,0 +1,269 @@
+"use client";
+
+import { Button } from "@/components/ui-components/button";
+import {
+  Checkbox,
+  CheckboxField,
+  CheckboxGroup,
+} from "@/components/ui-components/checkbox";
+import { Divider } from "@/components/ui-components/divider";
+import { Description, Field, Label } from "@/components/ui-components/fieldset";
+import { Heading, Subheading } from "@/components/ui-components/heading";
+import { Input } from "@/components/ui-components/input";
+import {
+  Radio,
+  RadioField,
+  RadioGroup,
+} from "@/components/ui-components/radio";
+import { Strong, Text } from "@/components/ui-components/text";
+import { Textarea } from "@/components/ui-components/textarea";
+import { FEATURE_FLAGS, isFlagEnabled } from "@/utils/flags";
+import Image from "next/image";
+import { notFound } from "next/navigation";
+import React, { useRef, useState } from "react";
+
+export default function Content() {
+  const flagEnabled = isFlagEnabled(FEATURE_FLAGS.JOB_POST_CREATE);
+  const fileInputRef = useRef<HTMLInputElement>(null);
+  const [imgUrl, setImgUrl] = useState<string | null>(null);
+
+  if (!flagEnabled) {
+    notFound();
+  }
+
+  return (
+    <form className="mx-auto max-w-4xl p-3 pt-8 sm:px-4">
+      <Heading level={1}>Post a job</Heading>
+      <Divider className="my-10 mt-6" />
+      <section className="grid gap-x-8 gap-y-6 sm:grid-cols-2">
+        <div className="space-y-1">
+          <Subheading level={2}>Company Logo</Subheading>
+          <Text>Square format is best</Text>
+        </div>
+        <Field>
+          <div className="flex items-center space-x-4">
+            {imgUrl && (
+              <Image
+                src={imgUrl}
+                width={80}
+                height={80}
+                alt="Company Logo"
+                className="rounded-[10px]"
+              />
+            )}
+            <div>
+              <Button
+                color="dark/white"
+                className="mt-3 rounded-md"
+                onClick={() => {
+                  fileInputRef.current?.click();
+                }}
+              >
+                Change Logo
+              </Button>
+              <Input
+                type="file"
+                id="file-input"
+                name="company-logo"
+                accept="image/png, image/gif, image/jpeg"
+                onChange={() => {}}
+                className="hidden"
+                ref={fileInputRef}
+              />
+              <Text className="mt-1 text-xs text-gray-500">
+                JPG, GIF or PNG. 1MB max.
+              </Text>
```

Comment:
We need a grey square as a placeholder for the image which will be uploaded.
Otherwise there will be layout shift when an image is added.
 
We need to be able to see the preview of the image to be uploaded. This preview only needs to be in state. Have a look at the settings page for an example of how this is handled.

Reply:
@John-Paul-Larkin Sure will do that way